assets/src/scripts/components/graph/view/construction.js

```diff
@@ -56,6 +54,11 @@ const drawWaterLevel = function (elem, elements, cursorWaterLevel) {
     if (!cursorWaterLevel) {
         return;
     }
+    // when there are no construction elements, water level height is wrong
```

Comment:
This bothers me. Should we try to fix the selector that calculates the water level height?

Reply:
We probably should make a follow up story because I feels like a another bug in the plotting.

Reply:
Looking at resolving my other magic values, I think it might be appropriate to set the water lower bound to always be bases on the graph height. I think the well is scaled to always fit.